apps/condo/domains/notification/adapters/smsAdapter.js

```diff
@@ -58,6 +64,125 @@ class SMSAdapter {
     }
 }
 
+class InstasentSms {
+
+    isConfigured = false
```

Comment:
You don't really need to save value of `validateConfig` inside class instance. You can just call `validateConfig` inside constructor of your adapter class.
```suggestion
```